views/js/qtiCreator/itemCreator.js

```diff
@@ -253,6 +253,28 @@ define([
                         });
 
                         this.item = item;
+
+                        if (config.properties) {
+                            if (config.properties.scalesPresets) {
+                                try {
+                                    const presets = JSON.parse(config.properties.scalesPresets);
+                                    item.data('scalePresets', presets);
+                                    item.data('scalesPresets', presets);
+                                } catch (e) {
+                                    console.warn('[ItemCreator] Failed to parse scale presets.', e);
```

Comment:
Please add in catch to just be sure nothing blows up :)
`item.data.scalesPresets = [];`
`item.data.scalePresets = [];`